### PR TITLE
BIP 75: Out of Band Address Exchange using Payment Protocol Encryption

### PR DESCRIPTION
**Pull request by techguy613**
Pull request for BIP75 as discussed on the mailing list

### CODE REVIEW

File: bip-0075.mediawiki [hunk 1]
```diff
@@ -0,0 +1,454 @@
+<pre>
+  BIP:     75
+  Title:   Out of Band Address Exchange using Payment Protocol Encryption
+  Authors: Justin Newton <justin@netki.com>
+           Matt David <mgd@mgddev.com>
+           Aaron Voisine <voisine@gmail.com>
+           James MacWhyte <macwhyte@gmail.com>
+  Status:  Draft
+  Type:    Standards Track
+  Created: 2015-11-20
```
Comment (luke-jr) [inline bip-0075.mediawiki]: Please don't add spaces to align the headers (leave the authors list as-is).

**Comment by techguy613** [comment]
Fixed the header and committed to our branch. PR should update soon.

**Comment by techguy613** [comment]
Hey @luke-jr, I'm not sure what's causing this git issue exactly. Per http://stackoverflow.com/questions/13480346/git-commits-left-behind-when-switching-back-to-a-branch-from-detached-head, this is a bug that was fixed in Git v1.7.11.5 and per Travis CI, their VM are using Git 1.8. Have you seen this before?

**Comment by luke-jr** [comment]
The git message isn't an issue, it's normal. The issue was that you had BIP 75 *after* BIP 8x in the README, and didn't use the comma-only formatting for the author list there. I fixed those for you and merged it.

**Comment by jmacwhyte** [comment]
Thanks @luke-jr!

**Comment by techguy613** [comment]
Thanks @luke-jr!

**Pull request merged**